### ISSUE TITLE
error: can't combine user with prefix, exec_prefix/home, or install_(plat)base

### ISSUE THREAD

**Issue by jeffhammond** [HPAC/TTC#5]
What am I doing wrong here?

```
TTC-github jrhammon$ python setup.py install --user 
running install
error: can't combine user with prefix, exec_prefix/home, or install_(plat)base
```

**Comment by springer13** [comment]
I need to look into this. It occurs on my Mac as well. I am using "python setup.py install" instead, this works for me. please let me know if this resolves the issue for you.

**Comment by jeffhammond** [comment]
Yeah, that works, except for the part where it perturbs `/usr/local/lib/python2.7/site-packages`.  I am obsessive about installation isolation, regardless of how antithetical this is to the Python Way.
